schema/incidents.hcl

```diff
@@ -396,13 +395,19 @@ table "incident_rules" {
     columns = [column.name]
   }
 }
+
 table "incidents" {
   schema = schema.public
   column "id" {
     null    = false
     type    = uuid
     default = sql("generate_ulid()")
   }
+  column "incident_id" {
+    null = false
+    type = varchar(10)
+    default = sql("format_incident_id(NEXTVAL('incident_id_sequence'))")
+  }
```

Comment:
This should have a unique clause too

Reply:
I've added it but I initially didn't do it because we're already using database sequence. But it does make sense to have it for absolute safety 👍